cmd/podman/containers/runlabel.go

```diff
@@ -61,11 +60,8 @@ func init() {
 	_ = flags.MarkHidden("opt1")
 	_ = flags.MarkHidden("opt2")
 	_ = flags.MarkHidden("opt3")
+	_ = flags.MarkHidden("pull")
 	_ = flags.MarkHidden("signature-policy")
-
-	if err := flags.MarkDeprecated("pull", "podman will pull if not found in local storage"); err != nil {
```

Comment:
Please keep the flag deprecated. Otherwise, users using it won't be notified ("Flag --pull has been deprecated, podman will pull if not found in local storage") on that it's a NOP.

Reply:
Added back